adapters/algorix/algorixtest/supplemental/no_imp_error.json

```diff
@@ -0,0 +1,13 @@
+{
+  "mockBidRequest": {
```

Comment:
Requests with empty `Imp` arrays are discarded upstream by prebid server core, no need to include this test.